software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcessWinRmDriver.java

```diff
@@ -49,15 +50,27 @@ public void start() {
 
     @Override
     public void install() {
+        // TODO: At some point in the future, this should probably be refactored to get the name of the machine in WinRmMachineLocation and set it as the hostname sensor
+        String hostname = null;
+        if (entity.getConfig(VanillaWindowsProcess.INSTALL_REBOOT_REQUIRED)) {
+            WinRmExecuteHelper checkHostnameTask = newScript("Checking hostname")
```

Comment:
@bostko @nakomis why do we retrieve this *before* executing the install command?

My understanding is that we added the hostname lookup to handle "Ability to perform installation on a Windows Blueprint which changes Computer Name and requires reboot." What will change the Computer Name? Do we expect it to be the "install.command"? If so, surely we should retrieve the hostname *after* executing the install command?